build.ps1

```diff
@@ -317,10 +331,10 @@ function Isolate-EnvVars([scriptblock]$Block) {
 
 function Invoke-VsDevShell($Arch) {
   $DevCmdArguments = "-no_logo -host_arch=$($HostArch.VSName) -arch=$($Arch.VSName)"
-  if ($WinSDKRoot) {
+  if (Test-Path script:WinSDKRoot) {
```

Comment:
What is this magic?  `script:WinSDKRoot`?  I assume it scopes it to the local variable rather than environment?